web.html

```diff
@@ -1,125 +1,224 @@
+            <div class="contain-images-2">
+                <img class="imagen-2" src="assets/img/imagen-2.jpg" alt="foto-2">
+                <img class="curve-blue" src="assets/img/curve-blue.png" alt="curve-blue">
+            </div>
+        </div>
+        </div>
+        <ul class="contain-social-media">
+            <li><a href=""><img src="assets/icons/icon-youtube.svg" alt="icon-youtube"></a></li>
+            <li><a href=""></a><img src="assets/icons/icon-linkedin.svg" alt="icon-linkedin"></li>
+            <li><a href=""><img src="assets/icons/icon-fbk.svg" alt="icon-fbk"></a></li>
+            <li><a href=""><img src="assets/icons/icon-twiter.svg" alt="icon-twiter"></a></li>
         </ul>
     </header>
     <main>
-        <section>
+        <section class="main-benefits">
             <h2>¿Por qué conviene YPF Ruta?</h2>
-            <div>
-                <div>
-                    <img src="" alt="">
+            <div class="contain-benefits">
+                <div class="benefit">
+                    <img src="assets/icons/icon-gasoil.svg" alt="icon-gasoil">
                     <h3>-20% en consumo de combustible</h3>
                     <p>Un manejo eficiente disminuye drásticamente el consumo de combustible.</p>
                 </div>
-                <div>
-                    <img src="" alt="">
+                <div class="benefit">
+                    <img src="assets/icons/icon-trucker.svg" alt="icon-trucker">
                     <h3>Reducí hasta un 20% los accidentes</h3>
                     <p>Ayuda a tus conductores a mejorar sus hábitos de conducción.</p>
                 </div>
-                <div>
-                    <img src="" alt="">
+                <div class="benefit">
+                    <img src="assets/icons/icon-map.svg" alt="icon-map">
                     <h3>Optimizá hasta en un 10% el uso de tu flota</h3>
                     <p>Planificá tus rutas y mejora tu operación logística.</p>
                 </div>
-                <div>
-                    <img src="" alt="">
+                <div class="benefit">
+                    <img src="assets/icons/icon-water.png" alt="icon-water">
                     <h3>Ahorrá hasta 3 litros por hora</h3>
                     <p>El motor encendido sin movimiento encrementa el consumo de combustible.</p>
                 </div>
-                <div>
-                    <img src="" alt="">
-                    <h3>20% menos en antenimiento</h3>
+                <div class="benefit">
+                    <img src="assets/icons/icon-key.svg" alt="icon-key">
+                    <h3>20% menos en mantenimiento</h3>
                     <p>Reduce los gastos de reparaciones y evita altos impactos económicos.</p>
                 </div>
             </div>
         </section>
-        <section>
-            <h2>Nuestros planes</h2>
+        <section class="main-plans">
+            <img src="assets/img/background-plans.png" alt="background-plans">
```

Comment:
En ingles así tendría sentido el nombre 😄 
```suggestion
            <img src="assets/img/background-plans.png" alt="plans-background">
```

Reply:
cambio realizado